inicheck/output.py

```diff
@@ -285,20 +287,21 @@ def print_non_defaults(ucfg):
                     # Grab the default, make it a string list
                     default_lst = mk_lst(mcfg[s][i].default)
                     str_default_lst = [
-                        str(kk).lower() for kk in default_lst if str(kk).lower() != 'none']
+                        str(kk).lower() for kk in default_lst if str(kk).lower() != 'none']  # noqa
```

Comment:
You should be able to split that in two lines and then the `#noqa` should not be required.